Taskfile.yml

```diff
@@ -25,11 +25,16 @@ includes:
     optional: true
     taskfile: ./bindings/go/ctf/Taskfile.yml
     dir: ./bindings/go/ctf
-
+  cli/configuration/v1:
+    optional: true
+    taskfile: ./cli/configuration/v1/Taskfile.yml
+    dir: ./cli/configuration/v1
 
 vars:
   GO_MODULES:
     sh: find {{ .ROOT_DIR }}/bindings/go -name go.mod -exec dirname {} \; | sed 's|{{ .ROOT_DIR }}/bindings/go/||'
+  GO_GENERATE:
```

Comment:
I think this requires appropriate fixes in github actions as well.

Reply:
We need the bindings in github actions just for task distribution, imo its totally fine to have a task that prints them for integration in github actions, then we dont need the scripts there.

Reply:
You can use a task instead of this line: https://github.com/open-component-model/open-component-model/blob/main/.github/workflows/ci.yml#L31 then you can unify it all

Reply:
Nice